### DISCUSSION TITLE
Version Info

### DISCUSSION THREAD

**Discussion by turtle-operator** [foxglove/community#651] [Q&A]
I don't remember exactly when, but with some recent version update the Foxglove Studio version info was removed (I think it previously was shown at the bottom of the settings panel, could that be?). From within the app, I would like to get an info about which application version is currently being used. Is there anything that I overlooked?

**Comment by defunctzombie** [comment]
Is this for the web app or desktop app?

In the desktop app the version info is available in `Help -> About` (or 🍎 -> About on Mac).

**Reply by turtle-operator** [reply]
Ideally both, but more importantly for the web app.
This is what I see in the desktop app on Ubuntu behind the help button:
![image](https://user-images.githubusercontent.com/109816930/223510626-a0f4301e-27be-4ef7-b9d6-40da687d2e89.png)
... no `About` page there.

**Comment by defunctzombie** [comment]
@jtbandes @2metres Did we change the location of the version information? I also recall it use to be in the help menu.

**Reply by jtbandes** [reply]
I've filed an internal tracking ticket FG-2364 to add the version info back.

**Comment by turtle-operator** [comment]
Has been resolved.
